Iris/Programs/TicTacToe.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <StartAction>Program</StartAction>
+    <StartProgram Condition="'$(ProgramW6432)'!=''">$(ProgramW6432)\dotnet\dotnet.exe</StartProgram>
+    <StartProgram Condition="'$(ProgramW6432)'==''">$(ProgramFiles)\dotnet\dotnet.exe</StartProgram>
+    <StartArguments>$(MSBuildThisFileDirectory)TicTacToe.dll</StartArguments>
+  </PropertyGroup>
+  
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.targets" />
```

Comment:
I thought line 1 `SDK="Microsoft.NET.Sdk"` does this.

Reply:
The problem is order -- if you don't have this Import statement, then these targets will get overwritten by the default target. So instead you need to include the default first, and then overwrite them.

Reply:
Its the same reason for it [here](https://github.com/microsoft/MIEngine/blob/4221ef54433d297aa88f49c027c7d8f9bc35f95e/src/MIDebugPackage/MIDebugPackage.csproj#L274) BTW.